api/controllers/console/workspace/workspace.py

```diff
@@ -71,6 +71,20 @@ def get(self):
         return WorkspaceService.get_tenant_info(tenant), 200
 
 
+class TenantRenameApi(Resource):
+    @setup_required
+    @login_required
+    @account_initialization_required
+    def put(self, tenant_id):
+        parser = reqparse.RequestParser()
+        parser.add_argument('name', type=str, required=True, location='json')
+        args = parser.parse_args()
+
+        TenantService.rename_tenant(current_user, tenant_id, args['name'])
```

Comment:
On this side, the controller layer needs to catch the exceptions thrown by the service layer and throw exceptions based on HTTPException to avoid internal errors.